_src/static/sass/base/_base.scss

```diff
@@ -0,0 +1,92 @@
+/* ============================
+-------------------------------
+ 1. Basic Setup
+-------------------------------
+============================ */
+
+
+/* ============================
```

Comment:
Please sync up this commenting style with the Sass/CSS styleguide's Section formatting - https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#table-of-contents

```scss
// ------------------------------
// #SETTINGS
// ------------------------------
// sections content goes here...
```